sonar-kotlin-plugin/src/main/resources/org/sonar/l10n/kotlin/rules/kotlin/S1206.html

```diff
@@ -0,0 +1,40 @@
+<p>In Kotlin you must override either both or neither of the <code>equals(Any?)</code> and <code>hashCode()</code> methods in order to keep the
```

Comment:
I found a couple of mistakes in the rule description - I updated the RSPEC PR, you should be able to simply update from there.